test/parallel/test-fs-watch-recursive.js

```diff
@@ -20,32 +18,42 @@ const testsubdir = fs.mkdtempSync(testDir + path.sep);
 const relativePathOne = path.join(path.basename(testsubdir), filenameOne);
 const filepathOne = path.join(testsubdir, filenameOne);
 
-const watcher = fs.watch(testDir, { recursive: true });
+if (!common.isOSX && !common.isWindows) {
+  common.expectsError(() => fs.watch(testDir, { recursive: true }),
+                      { code: 'ERR_FEATURE_UNAVAILABLE_ON_PLATFORM' });
+} else {
+
+  testWhenRecursiveIsImplemented();
+}
```

Comment:
I think @bnoordhuis intention was to have less churn. It is possible to just `return` or call `process.exit()` in the if statement. That way the rest of the code does not need any further change.

Reply:
done.